Call 3rd Party Web Service/Readme.md

```diff
@@ -1,39 +1,37 @@
 
-Many web services are open and public like the Earthquake API. Others require registration and API keys to restrict access to specific users and to limit the load on the service.
+Many web services are open and public like the Earthquake API. Others require registration and API keys that restrict access to specific users and limit the load on the service.
 
-To call a web service in the Web VI, we use the `HTTP Get` VI, passing in the URL of the service and receiving a string with the result of the call. There are examples of this in several VIs, including `GetRecentEarthquakes.gviweb`.
+To call a web service in a WebVI, this example uses the `HTTP Get` node to pass in the URL of the service and return a string with the result of the call. There are examples of this in several WebVIs, including `GetRecentEarthquakes.gviweb`.
 ![alt text](img/HTTPGet.png "Calling Web Service using HTTP Get")
 
-Web services return data in a variety of formats like JSON, XML, CSV, and YML. The USGS Earthquake service uses JSON, which LabVIEW is able to parse using `Unflatten from JSON`. We must specify the structure the data is expected to take and can also provide a path to limit the search to a specific part of the data. There are examples of this in several VIs, including `Get Earthquake Count.gviweb`.
+Web services return data in a variety of formats, such as JSON, XML, CSV, and YML. The USGS Earthquake service returns JSON, which LabVIEW can parse using the `Unflatten from JSON` node. To use `Unflatten from JSON`, you must specify the structure the data is expected to take and can provide a path to limit the search to a specific part of the data. There are examples of this in several WebVIs, including `Get Earthquake Count.gviweb`.
 ![alt text](img/UnflattenJSON.png "Parsing JSON using Unflatten from JSON")
 
-If the web service returns data in a format other than JSON, we can still extract it in the Web VI using functions from the String palette. There is an example of this in `Get Map URL.gviweb`.
+If the web service returns data in a format other than JSON, you can still parse it in the WebVI using String nodes. There is an example of this in `Get Map URL.gviweb`.
 
-Once the data has been retrieved and converted into LabVIEW types, we display it on the Top Level Web VI panel, `Main.gviweb`, using indicators from the panel palettes. This example uses a data grid to show all earthquakes and a URL Image to display the map of the surrounding area.
+Once the data has been retrieved and converted into LabVIEW data types, this example displays the data in indicators on the panel of the top-level WebVI, `Main.gviweb`. This example uses a data grid to show all earthquakes and a URL Image indicator to display a map of the surrounding area.
 
 ![alt text](img/Indicators.png "Wiring to indicators")
```

Comment:
Hey Jenny,

Since you're already updating this Readme, could you change the last 2 images to use capitalized ".PNG" in their filename (img/Indicators.PNG and img/UnflattenJSON.PNG) so they render in GitHub?